src/app/app-routing.module.ts

```diff
@@ -1,13 +1,21 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { MainListComponent } from './components/main-list/main-list.component';
 import { AdminFormCreateEventComponent } from './components/admin-form-create-event/admin-form-create-event.component';
 import { AdminFormManageEventComponent } from './components/admin-form-manage-event/admin-form-manage-event.component';
 
 
 const routes: Routes = [
+ 
+  {
+    path: 'mainList', component: MainListComponent
```

Comment:
lets call this route event/:id/present.  the :id is a dynamic route param and will be useful later.